src/transformers/models/hubert/modeling_hubert.py

```diff
@@ -628,24 +645,29 @@ def forward(
     ):
         attn_residual = hidden_states
         hidden_states = self.layer_norm(hidden_states)
-        hidden_states, attn_weights, _ = self.attention(
+        sa_output, sa_weights, _ = self.attention(
             hidden_states, attention_mask=attention_mask, output_attentions=output_attentions
         )
-        hidden_states = self.dropout(hidden_states)
-        hidden_states = attn_residual + hidden_states
-        hidden_states = hidden_states + self.feed_forward(self.final_layer_norm(hidden_states))
+        sa_output = self.dropout(sa_output)
+        sa_output = self.attention_adapters(sa_output, attn_residual)  # (bs, seq_length, dim)
```

Comment:
I'll change it to `sa_output = self.attention_adapters(hidden_states, attn_residual, None)` instead